### PR TITLE
Masterdiet 44

### PR DESCRIPTION
**Pull request by jfoder**

### CODE REVIEW

File: src/main/java/pl/agh/edu/master_diet/core/model/database/Achievement.java [hunk 1]
```diff
@@ -1,9 +1,17 @@
...
-import org.hibernate.annotations.Type;
-
-import javax.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.Column;
```
Comment (jakubowiczish) [inline src/main/java/pl/agh/edu/master_diet/core/model/database/Achievement.java:9]: About all these imports - maybe let's use import * ? You and @RafisSomeone keep replacing yourself with these imports :smile:

File: src/main/resources/liquibase.properties [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Liquibase properties to create diff between local database and JPA entities
+url=jdbc:postgresql://localhost:5432/master-diet-db
```
Comment (jakubowiczish) [inline src/main/resources/liquibase.properties:2]: Can you parametrize it in master-diet-secrets in order to be able to use that in production?

Comment (jakubowiczish) [reply src/main/resources/liquibase.properties:2]: That goes for url, username and password

File: src/main/java/pl/agh/edu/master_diet/core/model/database/Achievement.java (hunk 1 above)
Comment (jfoder) [reply src/main/java/pl/agh/edu/master_diet/core/model/database/Achievement.java:9]: What about IntelliJ import organization? We would have to disable this feature in commit dialog.

File: src/main/resources/liquibase.properties (hunk 2 above)
Comment (jfoder) [reply src/main/resources/liquibase.properties]: It turns out that these files are useless, liquibase uses properties from application.yml

File: src/main/java/pl/agh/edu/master_diet/core/model/database/Achievement.java (hunk 1 above)
Comment (jakubowiczish) [reply src/main/java/pl/agh/edu/master_diet/core/model/database/Achievement.java:9]: Yeah, so you can choose there an option to replace for import *

File: src/main/resources/liquibase.properties (hunk 2 above)
Comment (jakubowiczish) [reply src/main/resources/liquibase.properties]: Ok, cool

File: src/main/java/pl/agh/edu/master_diet/core/model/database/Achievement.java (hunk 1 above)
Comment (jfoder) [reply src/main/java/pl/agh/edu/master_diet/core/model/database/Achievement.java]: Ok, replaced with `import *`

**Comment by jakubowiczish** [comment]
If it works for create-drop strategy, I'll leave an approve, but for now, it requires some work

**Pull request merged**